src/Compilers/Core/MSBuildTask/CommandLineBuilderExtension.cs

```diff
@@ -235,9 +235,7 @@ internal void AppendSwitchIfNotNull
                                 else
                                 {
                                     // A boolean flag.
-                                    bool flagSet = false;
-
-                                    flagSet = Utilities.TryConvertItemMetadataToBool(parameter, metadataNames[i]);
+                                    bool flagSet = Utilities.TryConvertItemMetadataToBool(parameter, metadataNames[i]);
```

Comment:
This, and the next 3 differences, is just cleaning up other warnings. Seemed small enough to warrant getting the error list to empty (except messages).

Reply:
Turns out this was a silly comment, as its my changes that made these warnings appear, and they would have failed the build in CI anyway. Some kind of VS/MSBuild bug that the others didn't show up on my machine.